src/handlers/status_notifier_handler.cpp

```diff
@@ -47,9 +50,15 @@ void status_notifier_handler::on_request(
 		ss << "/" << id;
 	}
 
-	try {
-		helpers::curl_post(ss.str(), config_.port, params, config_.username, config_.password);
-	} catch (helpers::curl_exception &exception) {
-		logger_->critical("curl failed: {}", exception.what());
+	std::size_t retries = 10; // we come by this value using digital suction
```

Comment:
If this fails for any other reason than the API being unreachable, it will block all queued status notifications (that could otherwise proceed without delay). I'm not sure if that's a problem.